pwnlib/util/fiddling.py

```diff
@@ -416,7 +416,7 @@ def rol(n, k, word_size = None):
 def ror(n, k, word_size = None):
     """A simple wrapper around :func:`rol`, which negates the values of `k`."""
 
-    return ror(n, -k, word_size)
+    return rol(n, -k, word_size)
```

Comment:
Hm, I must have fixed that at some point and then forgotten about it.  It belongs in a separate PR but I'm too lazy :)